README.md

```diff
@@ -752,24 +777,59 @@ err := descopeClient.Management.AccessKey().Delete(context.Background(), "access
 
 ### Manage SSO Setting
 
-You can manage SSO settings and map SSO group roles and user attributes.
+You can manage SSO SAML or OIDC settings for a specific tenant.
```

Comment:
You can manage SSO (SAML or OIDC) settings for a specific tenant.

Reply:
done